src/Microsoft.AspNetCore.Mvc.TagHelpers/FormActionTagHelper.cs

```diff
@@ -194,7 +208,8 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 if (Route == null)
                 {
                     var urlHelper = UrlHelperFactory.GetUrlHelper(ViewContext);
-                    var url = urlHelper.Action(Action, Controller, routeValues);
+                    //add a null check for fragment?
```

Comment:
Is this a comment to reviewers? If yes, the answer is no.

If no, what value does the comment have for future developers and why does it lack a space after `//`?